doc/news/changes.h

```diff
@@ -385,6 +385,12 @@ inconvenience this causes.
 
 
 <ol>
+  <li> New: Added the class Polynomial for representation of polynomials. 
```

Comment:
Please say `Functions::Polynomial` so that doxygen automatically links it correctly.